packages/react-vapor/src/components/logoCard/LogoCard.tsx

```diff
@@ -7,28 +7,21 @@ import {slugify} from 'underscore.string';
 
 import {TooltipPlacement} from '../../utils';
 import {Badge, IBadgeProps} from '../badge/Badge';
-import {
-    CornerRibbon,
-    DEFAULT_CORNER_RIBBON_CONTAINER_CLASSNAME,
-    ICornerRibbonProps,
-} from '../cornerRibbon/CornerRibbon';
 import {Svg} from '../svg/Svg';
 import {Tooltip} from '../tooltip';
 
 export const DEFAULT_LOGO_CARD_CLASSNAME: string = 'logo-card';
 export const DEFAULT_LOGO_ICON: string = VaporSVG.svg.sourceCustom.name;
 export const DEFAULT_LOGO_ICON_CLASSNAME: string = 'icon';
 export const DEFAULT_LOGO_ICON_SIZE: string = 'mod-4x';
-export const DEFAULT_DISABLED_RIBBON_LABEL: string = 'Unavailable';
+export const DEFAULT_DISABLED_BADGE: string = 'Enterprise';

```

Comment:
I think I'd revert this to `'Unavailable'`

Reply:
Will it be mention in the tooltip that this source is for Enterprise plan? :thinking: Because I think it would be a tiny incentive to get the Enterprise plan :sunglasses: :wink:

Reply:
@aiheon still under R&D for now 😎